guildwars2/guild/general.py

```diff
@@ -43,14 +44,27 @@ class GeneralGuild:
             description='General Info about {0}'.format(guild_name),
             colour=await self.get_embed_color(ctx))
         data.set_author(name="{} [{}]".format(results["name"], results["tag"]))
-        data.add_field(name='Influence', value=results["influence"])
-        data.add_field(name='Aetherium', value=results["aetherium"])
-        data.add_field(name='Resonance', value=results["resonance"])
-        data.add_field(name='Favor', value=results["favor"])
+        data.add_field(
+            name="Influence",
+            value='{} {}'.format(
+                self.get_emoji(ctx, "influence"), results["influence"]))
+        data.add_field(
+            name="Aetherium",
+            value='{} {}'.format(
+                self.get_emoji(ctx, "aetherium"), results["aetherium"]))
+        data.add_field(
+            name="Resonance",
+            value='{} {}'.format(
+                self.get_emoji(ctx, "resonance"), results["resonance"]))
+        data.add_field(
+            name="Favor",
+            value='{} {}'.format(
+                self.get_emoji(ctx, "favor"), results["favor"]))
```

Comment:
This can be done with a for loop and a list of those currencies

Reply:
Changed to a for loop